bin/infer-boxes.jl

```
@@ -3,29 +3,40 @@
 import Celeste.ParallelRun: BoundingBox, infer_boxes
 import Celeste.Log
 
-
-const usage_info =
-"""
-Usage:
-  infer-boxes.jl <boxes_file> <out_dir>
-"""
-
-const stagedir = ENV["CELESTE_STAGE_DIR"]
-
 if length(ARGS) != 2
-    println(usage_info)
+    println("""
+        Usage:
+          infer-boxes.jl <boxes_file> <out_dir>
+
+        <boxes_file> format, one line per box:
+        <difficulty>	<#RCFs>	<#sources>	<ramin> <ramax> <decmin> <decmax>
+        """)
 else
     all_boxes = BoundingBox[]
-    sky_boxes = ARGS[1]
-    f = open(sky_boxes)
+    box_source_counts = Int64[]
+    boxes_file = ARGS[1]
+    f = open(boxes_file)
     for ln in eachline(f)
-        ss = split(ln, ' ')
-        ns = [parse(Float64, x) for x in ss]
-        bb = BoundingBox(ns...)
+        lp = split(ln, '\t')
+        if length(lp) != 4
```

maybe print an error message too...failing silently could be a problem